### PR TITLE
Using proper defaults

### PR DESCRIPTION
**Pull request by sophypal**
Since most of our apps will consume the `frost-app-bar`, I'm adding proper defaults to prevent it from being hidden.

This is a #MAJOR# change since this is ready to be consumed.

### CODE REVIEW

File: addon/styles/_variables.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
 // Notification base
 // -------------------------
 
+$notification-top: 50px !default;
```
Comment (sandersky) [inline addon/styles/_variables.scss]: Why are we using `!default`?

**Comment by sandersky** [comment]
:thumbsup:

File: addon/styles/_variables.scss (hunk 1 above)
Comment (sophypal) [inline addon/styles/_variables.scss]: Not sure.  I don't even know what it does but I was following the conventions in the file.  Also the `frost-theme` stuff does this as well.

Comment (sglanzer) [inline addon/styles/_variables.scss]: https://robots.thoughtbot.com/sass-default !default basically allows safe overrides that can be specified before (or after) our theme defaults

**Pull request merged**